### ISSUE TITLE
Loops in sequencer are not tight

### ISSUE THREAD

**Issue by yust1n** [kmatheussen/radium#1430]
### 1. Steps to reproduce the problem.

* Step 1: Start Radium.
* Step 2: Play a block with rshift
* Step 3: Set looppoints in sequencer to the same block and loop it

* I expected this to happen: Block should loop the same way as in Step 2

* This is what actually happened: Sequencerloop is 1 line too long ?!?!

Setting the grid to line and shorten the loop by 1 line plays fine.

**Comment by kmatheussen** [comment]
I can't reproduce that. Can you look at the attached song and modify it so that it exposes the bug?
[issue1430.rad.gz](https://github.com/kmatheussen/radium/files/14075852/issue1430.rad.gz)

**Comment by yust1n** [comment]
Found it ! I set Transport to Jack in Preferences->Sequencer a while ago. Didn't expect this to cause problems since Radium is the Jack Time Master. Thanks for confirming that I am the only who has/had this issue.

**Comment by kmatheussen** [comment]
Yeah,  there's no support for loops in Jack transport (as far as I know),
so you have to wait for jack to repositioning the play position, which can
take some  time.

**Comment by kmatheussen** [comment]
> http://users.notam02.no/~kjetism/radium/documentation/index.html?page=timing "Note that Radium always uses it's own timing. Radium never asks Jack Transport for timing." Hmmm ....

Note the definition of timing in the top of the page: "Timing is tempo and rhythm". It doesn't make sense for Radium to let other programs override tempo and rhythm in Radium. If you have set the tempo to 120BPM in Radium, that's what Radium is going to uses. Radium uses jack transport for starting and stopping playback.

**Comment by yust1n** [comment]
Thanks for your patience.
Ardour even disables looping completely when using jacktransport.
I just wonder why things like this not got fixed in jack2.
I have to lower my expectations :D

**Comment by yust1n** [comment]
Seems like the bug is "hunting" on me.
I removed all unnecessary from [testloop.rad.gz](https://github.com/kmatheussen/radium/files/14179958/testloop.rad.gz)
and did a small screenrecording [radiumloop.mp4.gz](https://github.com/kmatheussen/radium/files/14179983/radiumloop.mp4.gz).
Additionally I placed a small gap inbetween the blocks to assure that there is actually nothing been played
but still the first note of the next block gets triggered.
Trying to reproduce this behaviour in a new song failed unless the playback gets started via mousewheel.
For now I will just ignore it since I want to finish atleast one song with Radium :D

**Comment by kmatheussen** [comment]
Note to self: Before closing this issue, loop start/end positions needs to be fixed. They are positioned wrongly when using different samplerate than the song.

**Comment by kmatheussen** [comment]
Regarding the issue, it seems quite messy trying to fix this. The problem is that it takes some time before you get a message back from jack transport to stop playing, and then we have already played some stuff we don't want to play.

I think the better solution is to inform the user somehow that you get these types of problems if using jack transport.

**Comment by kmatheussen** [comment]
Oh, I'm sorry. The bug demonstrated in testloop.rad happens even if NOT using jack transport. Yeah, that shouldn't happen. :-)

**Comment by yust1n** [comment]
You could replicate it \o/
I already thought this happens only to me and ofcourse I wouldn't show this to you if I'd still use jacktransport.
